src/pytest_springfield/assertrepr_compare.py

```diff
@@ -0,0 +1,18 @@
+from _pytest.assertion.util import assertrepr_compare
+from springfield import Entity
+
+
+def pytest_assertrepr_compare(config, op, left, right):
+    """
+    Provide field-by-field comparisons if someone
+    uses `assert a==b` to compare two entities.
+    """
+    left_ent = isinstance(left, Entity)
+    right_ent = isinstance(right, Entity)
+    if not (left_ent or right_ent):
```

Comment:
left_ent and right_ent are booleans indicating left and/or right are instances of Entity, I just don't want to use this unless at least one of the two is an Entity.